### PR TITLE
[plugin.video.pcloud-video-streaming] 1.3.0

### PR DESCRIPTION
**Pull request by gdomenici**
### Description
v1.3.0 -- 21-Nov-2019
* Fixed issue #21: now the add-on runs with Python 3, used in Kodi 19 and higher, as well as Python 2, used in all previous Kodi versions.

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.pcloud-video-streaming/.vscode/.ropeproject/config.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# The default ``config.py``
```
Comment (Rechi) [inline plugin.video.pcloud-video-streaming/.vscode/.ropeproject/config.py]: Drop this file and also `plugin.video.pcloud-video-streaming/.vscode/.ropeproject/objectdb` from the Pull Request.

File: plugin.video.pcloud-video-streaming/resources/lib/test.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import pcloudapi
+from . import pcloudapi
```
Comment (Rechi) [inline plugin.video.pcloud-video-streaming/resources/lib/test.py]: It's great that you have a test for your add-on, but please remove it from the repo submission.

**Comment by gdomenici** [comment]
Closing PR; creating another one soon.

**Comment by Rechi** [comment]
There is no need to close this PR and create a new one, just update the PR branch (in your case helix from your repo) and it will show up here too.

**Comment by gdomenici** [comment]
OK, reopening... now squashed all commits and removed superfluous files.

**Pull request closed without merging**